web/src/main/resources/log4j.properties

```diff
@@ -9,7 +9,10 @@ log4j.logger.org.hibernate=INFO
 log4j.logger.com.trg=INFO
 
 # File appender
-log4j.appender.fileAppender=org.apache.log4j.RollingFileAppender
+log4j.appender.fileAppender=org.apache.log4j.rolling.RollingFileAppender
+log4j.appender.fileAppender.rollingPolicy=org.apache.log4j.rolling.TimeBasedRollingPolicy
+log4j.appender.fileAppender.rollingPolicy.FileNamePattern=/tmp/mapstore.%d.log.gz
 log4j.appender.fileAppender.layout=org.apache.log4j.PatternLayout
+log4j.appender.fileAppender.Append = true
 log4j.appender.fileAppender.layout.ConversionPattern=%p   %d{yyyy-MM-dd HH:mm:ss.SSS}   %C{1}.%M() - %m %n
-log4j.appender.fileAppender.File=${catalina.base}/logs/GeOrchestra.log
\ No newline at end of file
+log4j.appender.fileAppender.File=/tmp/mapstore.log
```

Comment:
Could you use `${java.io.tmpdir}` or something similar to make it work also locally on windows machines?

Reply:
geOrchestra doesnt target windows anyway, and i dunno if `java.io.tmpdir` has a default value if unset. `/tmp/` is for consistency with other georchestra modules (ie https://github.com/georchestra/datadir/blob/master/console/log4j/log4j.properties#L24 https://github.com/georchestra/datadir/blob/master/geonetwork/log4j/log4j.xml#L6 https://github.com/georchestra/datadir/blob/master/mapfishapp/log4j/log4j.properties#L9 and so on), as we're sure tomcat can write there. In production i use `/srv/log/tomcat` but that's because my ansible infra creates that dir with correct ownership.